### PR TITLE
 STL Algorithms In C++

### PR DESCRIPTION
**Pull request by VARADSP**
This pull request fixes #(issue number)

### CODE REVIEW

File: sortcom.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
+//sorts array of strings with user written comparison function
```
Comment (t2013anurag) [inline sortcom.cpp]: can you move the file inside sorting/sort_strings

File: sortcom.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
+//sorts array of strings with user written comparison function
+#include<iostream>
+#include<string.h>
+#include<functional>
+#include<algorithm>
+
```
Comment (t2013anurag) [inline sortcom.cpp]: remove all extra lines.

File: sortcom.cpp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+{
+    return name=="Don";
+}
+double in_to_cm(double s)
+{
+    return (s * 2.54);
+
+}
+
+
```
Comment (t2013anurag) [inline sortcom.cpp]: remove.

File: sortcom.cpp [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    return 0;
+
+
+}
+bool alpha_comp(char *s1,char *s2)
+{
+    return (strcmp(s1,s2)<0)?true:false;
+}
+
+
```
Comment (t2013anurag) [inline sortcom.cpp]: add sample input and output in the comment.

File: sortcom.cpp (hunk 1 above)
Comment (VARADSP) [reply sortcom.cpp]: how to move it please tell me its my first contribution

Comment (t2013anurag) [reply sortcom.cpp]: Sure, you need to just copy the file to sorting/sort_strings directory and then delete this file.

finally 
```git pull -r origin master
git add .
git commit -m "moved files"
git push origin master```

**Comment by VARADSP** [comment]
OK thank you very much

**Comment by VARADSP** [comment]
Input:
//sorts these string values
string names1[] = {"George","Penny","Estelle","Don","Mike","Bob"};
//converts these inches to centimetres
 double inches[] = {3.5,6.4,6,7,8};

Ouput:
 8.89    16.256  15.24   17.78   20.32   5.34025e-317    22.5806
Don is element 3 on the list
Bob
Don
Estelle
George
Mike
Penny

--------------------------------
Process exited after 0.004728 seconds with return value 0
Press any key to continue . . .

**Comment by VARADSP** [comment]
I am not able to move this file into sorting directory help me
and I cant find sort_string directory

**Comment by t2013anurag** [comment]
@VARADSP - create a new directory inside sorting named sort_strings

**Comment by VARADSP** [comment]
I think I moved the file in sort_strings

**Pull request closed without merging**